src/components/CustomerDetails/CustomerDetails.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import CustomerDetails from '../CustomerDetails';
+
+describe('CustomerDetails', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = mount(
+        <CustomerDetails/>
+      );
+  });
+
+  it('renders a customer form first heading ', () => {
+    expect(wrapper.find("h3").at(0).text()).toEqual("Shipping Details");
+  });
+
+  it('renders a customer form fourth label ', () => {
+    expect(wrapper.find("FormItem").at(3).text()).toEqual("City");
+  });
+
+  it('renders a customer form second step', () => {
+    expect(wrapper.find("Step").at(1).text()).toEqual("Items")
+  })
+
+  it('contains two postcodes for shipping + billing', () => {
```

Comment:
There is a tradeoff here between "is this component on :fire:" vs testing stuff that is of low value.

I'm wondering whether a test that fills out the entire form, submits and checks it was sent to a remote service would test that all the fields are present indirectly?